app/src/main/java/com/github/leuludyha/ibdb/presentation/navigation/BottomToolbar.kt

```diff
@@ -18,7 +18,7 @@ fun BottomToolbar(navController: NavHostController?, defaultSelection: Int = 0)
     val tabs = listOf(
         TabDescriptor.Home,
         TabDescriptor.Search,
-        TabDescriptor.Maps,
+        TabDescriptor.Collection,
         TabDescriptor.Profile
```

Comment:
Why did you remove the maps screen? This is something we should rather talk as a group, instead of changing it this way, since it is supposed to be one of the main features of the app and has nothing to do with this pull's changes.